docs/coding-guidelines.md

```diff
@@ -57,6 +57,8 @@ For example the following are correct:
     string fruit = null; // can't use "var" because the type isn't known (though you could do (string)null, don't!)
     const string expectedName = "name"; // can't use "var" with const
     FruitFlavor flavor = fruit.GetFlavor();
+    var nugetVersion = NuGetVersion.Parse("1.0.0");
+    var solutionManager = ComponentModel.GetService<ISolutionManager>();
```

Comment:
it's not actually clear to me, in this second instance, that it would -definitely- be an `ISolutionManager`, even with that type parameter there.

Reply:
Can you elaborate why?

Reply:
It's possible I'm just not familiar with a common pattern here, but it's just not obvious to me. Reading this line, I would expect to get some kind of service back, as determined by `ISolutionManager`, not an actual `ISolutionManager` instance.

Reply:
What's your thought on the following? 

```cs
	var empty = Enumerable.Empty<int>();
```

Good or confusing?

Reply:
I think this is better -- `Enumerable.Empty` is a core language feature, yeah? That's a bit different from library stuff, which could really be anything.

Reply:
ok, I'l remove the service example then.